src/main/java/gregtech/api/block/VariantActiveBlock.java

```diff
@@ -82,6 +82,7 @@ protected BlockStateContainer createBlockState() {
     @Override
     public IExtendedBlockState getExtendedState(IBlockState state, IBlockAccess world, BlockPos pos) {
         IExtendedBlockState ext = (IExtendedBlockState) state;
+        //ACTIVE_BLOCKS.putIfAbsent(Minecraft.getMinecraft().world.provider.getDimension(), new ObjectOpenHashSet<>());
```

Comment:
This shouldn't be commented out I think. Otherwise I still get the rendering error with invisible blocks